.github/workflows/ci.yml

```diff
@@ -32,19 +32,26 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v5
+        uses: actions/setup-python@9e62be81b28222addecf85e47571213eb7680449
         with:
           python-version: ${{ matrix.python-version }}
           architecture: ${{ matrix.architecture }}
 
+
+      - if: ${{ endsWith(matrix.python-version, 't') }}
+        run:
+          pip install --pre -i https://pypi.anaconda.org/scientific-python-nightly-wheels/simple numpy
+          pip install pytest-run-parallel
+          echo "PARALLEL_THREADS=4" >> "$GITHUB_ENV"
+
       - name: Install
         run: |
           pip install .
 
       - name: Test with pytest
         run: |
           pip install pytest
-          pytest --pyargs bottleneck
+          pytest --parallel-threads=$PARALLEL_THREADS --pyargs bottleneck
```

Comment:
This doesn't look correct, since `pytest-run-parallel` is only installed for cp313t. On other Python versions, this should result in:
```
pytest: error: unrecognized arguments: --parallel-threads=
```

It's annoying that CI doesn't run on this PR, but can you try running the CI job on your own fork to make sure it passes? (same for wheel builds)